apps/perf-test/tasks/perf-test.js

```diff
@@ -95,31 +140,31 @@ module.exports = async function getPerfRegressions() {
     });
   }
 
-  const iterationsArg = Number.isInteger(argv().iterations) && argv().iterations;
+  const iterationsArgv = /** @type {number} */ (argv().iterations);
```

Comment:
Another way to do this (which I think adds automatic type inference) is to define these as yargs options. This also lets you enable automatic usage help.
```
const yargs = require('yargs');
yargs.option('iterations', { type: 'string', default: 5000, description: 'whatever' }
  .option(...)
  .help() // enable --help
  .argv;
```

Reply:
As this is a just script, I made use of argv() from just scripts, instead. I don't see a lot of existing convention around help, at least looking through our source base.

Reply:
Oh right, I forgot about that

Reply:
I was thinking about yargs options because I was just working on another script (not run via just) where using them made sense.